lang/cs/Org.Apache.REEF.Evaluator.Tests/ContextRuntimeTests.cs

```diff
@@ -37,13 +42,7 @@ public sealed class ContextRuntimeTests
         public void TestContextEvents()
         {
             const string hello = "Hello!";
-            var contextConfig = ContextConfiguration.ConfigurationModule
-                .Set(ContextConfiguration.Identifier, "ID")
-                .Set(ContextConfiguration.OnContextStart, GenericType<ContextEventHandler>.Class)
-                .Set(ContextConfiguration.OnContextStop, GenericType<ContextEventHandler>.Class)
-                .Set(ContextConfiguration.OnMessage, GenericType<ContextEventHandler>.Class)
-                .Build();
-            
+            var contextConfig = GetContextEventHandlerContextConfiguration();
             var injector = TangFactory.GetTang().NewInjector();
 
             var handler = new ContextEventHandler();
```

Comment:
why create a handler here? When you pass contextConfig into ContextRuntime and fork an _contextInjector, the handler object will be instantiated, right? ,